soa/service/remote_credential_provider.cc

```diff
@@ -1,188 +0,0 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
```

Comment:
OK, that's fine.  It was designed as something that is entirely orthogonal: you add an environment variable, you get something that will be called for credentials and it doesn't affect any other functionality nor is it enabled by defuault.  Like I said, escape hatch.